dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/logging/source/AccessLogSource.java

```diff
@@ -592,12 +594,21 @@ private static SetterFormatter createSetterFormatter(Configuration config, Forma
 
             //Include the requestFirstLine for Telemetry to be used as the body where applicable
             fieldSetters.add((ald, alrd) -> ald.setRequestFirstLine(AccessLogFirstLine.getFirstLineAsString(alrd.getResponse(), alrd.getRequest(), null)));
+            fieldSetters.add((ald, alrd) -> ald.setRequestHeader("traceparent", AccessLogRequestHeaderValue.getHeaderValue(alrd.getResponse(), alrd.getRequest(), "traceparent")));
```

Comment:
Should put the `traceparent` and other propagators in a Constants class of some sort, instead of hardcoding, for easier maintenance in the future....